### PR TITLE
Milestone4:Desktop version of the project

### PR DESCRIPTION
**Pull request by ismayito**
#Desktop version and responsiveness
In this milestone; We added the following.
- The desktop version on the project and also made it responsive

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -106,13 +114,26 @@ body {
   padding-bottom: 11.4rem;
 }
 
+.desktop-project {
+  display: flex;
+  flex-direction: column;
+}
```
Comment (levy002) [inline style.css:120]: - [ ] Kindly make sure to align the project information in its card correctly in order to match the Figma design as much as possible.

  - ### YOUR DESIGN
  
    image

  - ### FIGMA DESIGN
  
   image

File: style.css [hunk 2]
```diff
@@ -360,3 +389,178 @@ body {
...
+  .lang-desk {
+    display: inline;
+  }
+
+  /* About me section */
+  .about-me {
+    display: grid;
+    grid-template-columns: 1fr 1fr;
+    padding-top: 14.2rem;
+  }
```
Comment (levy002) [inline style.css:544]: - [ ] Kindly make sure to provide enough padding on both sides of the `About-me` section in order to match the Figma design as much as possible. 

- [ ] Kindly make sure your `Get my resue` button width is the same as the one Figma design.

  - ### YOUR DESIGN
  
    image

  - ### FIGMA DESIGN
  
    image

File: style.css [hunk 3]
```diff
@@ -360,3 +389,178 @@ body {
...
+    margin-top: 0;
+  }
+
+  .social-about {
+    margin-bottom: 3.4rem;
+  }
+
+  .lang {
+    flex-direction: row;
+  }
```
Comment (levy002) [inline style.css]: - [ ] Kindly make sure the `Languages` card has the same height and width as the one on the Figma design.

   - ### YOUR DESIGN
   
     image

   - ### FIGMA DESIGN
   
    image

**Review by levy002 — CHANGES_REQUESTED** [review]
## Hello @ismayito   , :wave:
### I'm Levy, your PR reviewer

Good job so far :+1: 
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights
- Professional PR :heavy_check_mark: 

## Status: REQUIRED CHANGES ♻️
_Check the comments under the review._

- [ ] Kindly make sure to add enough padding on both sides of the navigation bar in order to match the provided Figma design.

   - ### YOUR DESIGN
   
    image

   - ### FIGMA DESIGN
    
    image


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @levy002 in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by mahbubul14 — APPROVED** [review]
# Project Approved :trophy: :tada: 

Hi @ismayito, 

Your project is complete!  You have done it exceptionally well. it's time to merge it :100: 
Congratulations! 🎉 

### To Highlight :+1: 

:heavy_check_mark: No linter error.
:heavy_check_mark: Descriptive readme file.
:heavy_check_mark: Well-implemented suggestions.

### Optional suggestions :arrow_down_small: 

Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better.

- [ ] The commit message should be written in the imperative tense and the first letter should also be capitalized. `Ex: "Fix bug" instead of "fix bug".` [For more details...](https://www.freecodecamp.org/news/level-you-up-to-awesome-commit-messages-a85558cb90e8)

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification. You can also connect with me on [slack](https://microverse-students.slack.com/team/U01J39GL614)

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Pull request merged**